src/lib/merkle_mask/masking_merkle_tree.ml

```diff
@@ -22,29 +22,25 @@ module Make (Inputs : Inputs_intf.S) = struct
     type t = unit Async.Ivar.t
   end
 
-  type maps_t = Inputs.maps_t =
-    { accounts : Account.t Location.Map.t
-    ; token_owners : Account_id.t Token_id.Map.t
-    ; hashes : Hash.t Addr.Map.t
-    ; locations : Location.t Account_id.Map.t
-    ; non_existent_accounts : Account_id.Set.t
-    }
+  type maps_t = Inputs.Mask_maps.t
```

Comment:
Maybe we could consider removing the `type maps_t` now, given it's in the inputs?

Check all of the places where we introduced the `maps_t` and see if just having it in inputs would have been sufficient